test/verify/check-multi-machine

```diff
@@ -311,7 +313,7 @@ class TestMultiMachine(MachineCase):
         b.logout()
 
         # Bad host key
-        m.execute("echo '10.111.113.2 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDgPMmTosSQ4NxMtq+aL2NKLC+W4I9/jbD1e74cnOKTW' > /home/admin/.ssh/known_hosts")
+        m.execute("echo '10.111.113.2 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDgPMmTosSQ4NxMtq+aL2NKLC+W4I9/jbD1e74cnOKTW' > /etc/ssh/ssh_known_hosts")
```

Comment:
Wait, why did this switch back to the global file? Surely when there's a bad hostkey in ~/.ssh/_known_hosts it should fail too.

Reply:
This is testing the bastion host scenario, and cockpit-ssh is not actually running as "admin".